epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityTracker.java

```diff
@@ -100,6 +103,19 @@ public void setOnChangedEnabled(boolean enabled) {
     onChangedEnabled = enabled;
   }
 
+  /**
+   * Set the threshold of percentage visible area to identify the partial impression view state.
+   *
+   * @param thresholdPercentage Percentage of visible area of an element. 0..100. A value of 0
+   *                            effectively disables
+   *                            {@link VisibilityState#PARTIAL_IMPRESSION_VISIBLE} events.
+   */
+  public void setPartialImpressionThresholdPercentage(
+      @IntRange(from = 0, to = 100) int thresholdPercentage
```

Comment:
I am wondering if `0` should be considered as `null`. Is it even a valid threshold for this use case?

Reply:
See this discussion 
https://github.com/airbnb/epoxy/pull/973#discussion_r426092299